src/shared/ui/Button/Button.tsx

```diff
@@ -0,0 +1,14 @@
+import { FC } from 'react';
+import MaterialButton, { ButtonProps as MaterialButtonProps } from '@mui/material/Button';
+
+interface ButtonProps extends MaterialButtonProps {}
```

Comment:
Let's remove extends MaterialButtonProps and just add children and onClick fields to the interface